### PR TITLE
describe about separation of branch

### PR DESCRIPTION
**Pull request by kinichiro**
- add notification for translators to submit PR to stable branch in README.adoc
- add "branch relations, working rules" section in docs-versioning.adoc

This PR is relates to https://github.com/KiCad/kicad-doc/issues/408 .
A diagram may not needed.
Any comments would be highly appreciated.

### CODE REVIEW

File: docs-versioning.adoc [hunk 1]
```diff
@@ -60,6 +56,62 @@ documentation releases to be updated independent of the KiCad release
...
+                |                      |
+                V                      V
+<stable>       [#]------$-----        [#]  -  -  -  - {translators}
+               <4.0>    4.0.3         <5.0>
+```
+
+On *master* branch:
+
+- Writers update English .adoc files and related English menu screenshots.
+- Translation works are not accepted here, since those are invalidated soon
```
Comment (nickoe) [inline docs-versioning.adoc]: I don't think we should blindly reject translations here if people want to translate, but we still want to inform them that the master reference document is volitile, so translations will probably be invalidated quickly.

File: docs-versioning.adoc [hunk 2]
```diff
@@ -60,6 +56,62 @@ documentation releases to be updated independent of the KiCad release
...
+  by updating .adoc files.
+
+On stable branch (e.g. *4.0*):
+
+- Translators create Localized .po files and related localized menu screenshots
+  for stable docs.
+- Basically, original English .adoc files are not modified here, since it
+  invalidates translation works for the stable release doc.
```
Comment (nickoe) [inline docs-versioning.adoc]: I think we should still allow them to be changes, if forexample there are typos or the like. We should just require that the translations are not invalidated by it. Usually just by also fixing the english typo in the po files.

**Comment by nickoe** [comment]
@kinichiro I have made some inline comments to your change. If you respond to the comment, I am fine with merging this.

**Comment by kinichiro** [comment]
@nickoe I made small changes to docs-versioning.adoc,
could you check the commit https://github.com/KiCad/kicad-doc/pull/411/commits/8552da3f9ebe63d5c9710420e1465ba759c0a82f above, please.
Thanks.

**Comment by nickoe** [comment]
Looks good to me, thanks!

**Pull request merged**